test/Concurrency/actor_call_implicitly_async.swift

```diff
@@ -0,0 +1,177 @@
+// RUN: %target-typecheck-verify-swift -enable-experimental-concurrency
+// REQUIRES: concurrency
+
+actor class BankAccount {
+
+  private var curBalance : Int
+
+  private var accountHolder : String = "unknown"
+
+  // expected-note@+1 2 {{mutable state is only available within the actor instance}}
+  var owner : String {
+    get { accountHolder }
+    set { accountHolder = newValue }
+  }
+
+  init(initialDeposit : Int) {
+    curBalance = initialDeposit
+  }
+
+  // NOTE: this func is accessed through both async and sync calls.
+  // expected-note@+1 {{only asynchronous methods can be used outside the actor instance; do you want to add 'async'?}}
```

Comment:
The text of this note is incorrect now, and the "add `async`" Fix-It isn't the right fix. It's a bit jargon-y, but what about something like?

```
method called on an actor other than `self` will be invoked asynchronously
```

Reply:
Good catch! I have this note being produced:

```
calls to instance method 'balance()' from outside of its actor context are implicitly asynchronous
```

I tried to avoid "actor-isolated" here to reduce the jargon.

Reply:
"Actor-isolated" or "isolated to an actor" is used in more than a dozen different diagnostics, so skipping it here isn't really going to avoid that jargon. This is a case where we will need to decide on the terms we should use, which might mean teaching our users about "actor-isolated" as a fundamental concept, and then take a pass through the diagnostics to make them all use consistent terminology.